sdk/azidentity/azidentity.go

```diff
@@ -164,8 +169,9 @@ func newDefaultMSIPipeline(o ManagedIdentityCredentialOptions) azcore.Pipeline {
 	var statusCodes []int
 	// retry policy for MSI is not end-user configurable
 	retryOpts := azcore.RetryOptions{
-		MaxTries:   5,
+		MaxRetries: 5,
```

Comment:
Note this changed from tries to retries.  Leaving this as 5 will mean a total of 6 tries.  Is this expected, or should the value change to 4?

Reply:
Yes, you're right this needs to be changed to 4.